Services/CO.CDP.Organisation.WebApi/Api/Organisation.cs

```diff
@@ -111,10 +119,15 @@ await useCase.Execute(organisationId)
             });
         app.MapDelete("/organisations/{organisationId}", (Guid organisationId) =>
             {
+                if (!_organisations.ContainsKey(organisationId))
+                {
+                    return Results.NotFound();
+                }
                 _organisations.Remove(organisationId);
                 return Results.NoContent();
             })
             .Produces(204)
+            .Produces(404)
```

Comment:
Should 404 be returned from Delete?
https://stackoverflow.com/questions/6439416/status-code-when-deleting-a-resource-using-http-delete-for-the-second-time

Reply:
I will remove it - in our case it might not the case - it only happens in case user try to delete the same resource twice. but in the UI we should navigate away after the first deletion. I will remove this.